recipe/meta.yaml

```diff
@@ -31,6 +30,7 @@ requirements:
     - meson-python                           # [build_platform != target_platform]
     - ninja                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
     - {{ compiler('cxx') }}
   host:
     - python
```

Comment:
For a few lines below, should `{{ pin_compatible('numpy') }}` be change to just `numpy` ?

Reply:
Thanks for catching that, I overlooked it while doing it manually. It should be removed.